How to install using git repository?

Getting errors following instructions at [INSTALL ](https://github.com/brailcom/speechd/blob/master/INSTALL)

```
./configure: line 12297: syntax error near unexpected token `0.40.0'
./configure: line 12297: `IT_PROG_INTLTOOL(0.40.0)'
~/speechd-master$ make all
make: *** No rule to make target 'all'.  Stop.
```

Le 18/02/2018 à 21:37, guest271314 a écrit :
> Getting errors following instructions at INSTALL 
> <https://github.com/brailcom/speechd/blob/master/INSTALL>
> 
> |./configure: line 12297: syntax error near unexpected token `0.40.0' 
> ./configure: line 12297: `IT_PROG_INTLTOOL(0.40.0)' ~/speechd-master$ 
> make all make: *** No rule to make target 'all'. Stop. |
> 

What is your build environment?

Best regards.

--- a/INSTALL
+++ b/INSTALL
@@ -14,7 +14,7 @@ You will need these components to compile Speech Dispatcher:
         (http://www.python.org)
 
 Only if you are building from Git, you also need these:
- - automake 1.11+, autoconf, pkg-config and libtool
+ - automake 1.11+, autoconf, pkg-config, libtool and gettext
 
 We recommend to also install these packages:
  - PulseAudio (http://www.pulseaudio.org)

